src/vivarium/framework/results/stratification.py

```diff
@@ -66,5 +66,5 @@ def __call__(self, population: pd.DataFrame) -> pd.DataFrame:
         return population
 
     @staticmethod
-    def _default_mapper(pop: pd.DataFrame) -> str:
+    def _default_mapper(pop: pd.DataFrame) -> Union[pd.Series, str]:
```

Comment:
This can't ever be `str`. The `axis=1` argument ensures we always return a Series (or a DataFrame if `pop` has more than 1 column). 

I've forgotten if this is enforced, but it's not permissible to use the default mapper if the pop passed in has more than one column. Mappers always have to return a Series.